bolt-modules/boltlib/lib/puppet/functions/set_resources.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+# Sets one or more resources on a target.
```

Comment:
I think it's important to note that this doesn't apply the resource to the target.